docs/guide/tutorial-8-the-supervisor.md

```diff
@@ -1,9 +1,5 @@
 ## Tutorial 8: the Supervisor (30 Minutes)
 
-A [Supervisor](../reference/supervisor.md) is a special type of [Robot](../reference/robot.md) which has additional powers.
-In fact, any [Robot](../reference/robot.md) can be turned into a supervisor simply by setting its field named `supervisor` to TRUE.
-A [Supervisor](../reference/supervisor.md) can modify the environment by adding or removing nodes to the scene, it can change their properties by modifying the values of parameters in a programmatic way, allowing for instance to move or setup a robot a certain way, it can start the creation of movies, animations and, last but not least, thanks to its unlimited access it can be used to acquire measurements about the state of the simulation as well as track its evolution.
-
 This tutorial will explore how to achieve some of these tasks using a [Supervisor](../reference/supervisor.md).
```

Comment:
This phrase doesn't make much sense without the rest, the tutorial should still have an intro that explains what is being done there

Reply:
I think your description is very clear and should go in the reference manual, so I adapted the tutorial to redirect to the reference manual. what do you think?

Reply:
> What do you think about this?

yes it is better. I have just change the last sentence, what do you think?